### PR TITLE
Implement new Model concept

### PR DESCRIPTION
**Pull request by dermrsimon**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
The current model concept is badly planned and not usable. A new one is needed. #3 

**What is the new behavior (if this is a feature change)?**
A thoughtful model of Pages, Forms and Objects is now implemented.

### CODE REVIEW

File: charm/reports/models.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      FieldPanel('button_link'),
+      PageChooserPanel('button_page')
+    ]
+
+    def __str__(self):
+      return self.button_title
+
+class _S_ParagraphBlock(blocks.StructBlock):
+    statement = blocks.CharBlock(null=True, blank=True, help_text="Statement")
+    paragraph = blocks.RichTextBlock(null=True, blank=False, help_text="Manifest paragraph", features=['bold', 'italic', 'underline', 'strikethrough', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'blockquote', 'ol', 'ul', 'hr', 'embed', 'link', 'superscript', 'subscript', 'document-link', 'image', 'code'], classname="full")
```
Comment (Aichnerc) [inline charm/reports/models.py]: The maximum length per line is 119 characters.

Comment (dermrsimon) [reply charm/reports/models.py]: Affirmative.

**Comment by dermrsimon** [comment]
ok i changed it

**Comment by Aichnerc** [comment]
image

**Pull request merged**